.env.development

```diff
@@ -1,5 +1,6 @@
 DQT_API_URL=https://preprod-teacher-qualifications-api.education.gov.uk/
 DQT_API_KEY=testkey
+GOVUK_NOTIFY_API_KEY=test
```

Comment:
I wrote this in the README about getting your own Notify key:

https://github.com/DFE-Digital/find-a-lost-trn#notify

Then you'd actually be able to open the Notify dashboard and see the emails that were sent out. If you use a dummy key, the mailer will silently fail right?

So I guess the question here is whether for development we want to blow up and ask that the user sets up a local test key, or silently fail.

What's your preference?

Reply:
I have a test key as well. 

Personally, I don't think we should have the development .env in version control. I end up stashing changes to it for every new branch I work on.

Reply:
You can have `.env.development.local`, does that work?

Reply:
Yeah, ok. I'll use that.

Reply:
Cool, then yeah I think we can keep `.local` ones out of version control and the `.env.development` can stay in as that fits the suggested use here https://github.com/bkeepers/dotenv#what-other-env-files-can-i-use